pipeline/scripts/5/1/tier-1/test-upgrade-4x-to-5x-rgw-tests.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+echo "Beginning RHCS 4x to 5x upgrade testing with RGW tests."
```

Comment:
Can we move this test to a different folder? Preferably to `tier-3` folder. All the stages are executed against every build. The reason behind the request is that we already have two many tests in this folder ... moving it out would improve the feedback loop.

Reply:
Done